src/styles/utils/function.scss

```diff
@@ -0,0 +1,30 @@
+@import "variables.scss";
+
+@for $index from 1 through 31 {
+  .calendar-day:nth-child(#{$index})::before {
+    content: "#{$index}";
+  }
+}
+
+$weekdays:
```

Comment:
move into variables

Reply:
I thought it is another block, it is easer to work with it separately